src/include/stir/recon_buildblock/LogcoshPrior.h

```diff
@@ -125,9 +125,10 @@ class LogcoshPrior:  public
                                        const DiscretisedDensity<3,elemT> &current_image_estimate);
 
     //! compute Hessian
-    void compute_Hessian(DiscretisedDensity<3,elemT>& prior_Hessian_for_single_densel,
-                         const BasicCoordinate<3,int>& coords,
-                         const DiscretisedDensity<3,elemT> &current_image_estimate);
+    virtual Succeeded
+    compute_Hessian(DiscretisedDensity<3,elemT>& prior_Hessian_for_single_densel,
+                    const BasicCoordinate<3,int>& coords,
+                    const DiscretisedDensity<3,elemT> &current_image_estimate) const;
 
     //! Compute the multiplication of the hessian of the prior multiplied by the input.
```

Comment:
let's correct this one as well
```suggestion
    //! Compute the multiplication of the hessian of the prior (at \c current_estimate) and the \c input.
```